tests/ecdsa.js

```diff
@@ -22,6 +28,34 @@ function test (binding) {
       t.same(binding.sign(m, d), expected, `sign(${f.m}, ...) == ${f.signature}`)
     })
 
+    fecdsa.extraEntropy.forEach((f) => {
+      const d = Buffer.from(f.d, 'hex')
+      const m = Buffer.from(f.m, 'hex')
+      const expectedSig = Buffer.from(f.signature, 'hex')
+      const expectedExtraEntropy0 = Buffer.from(f.extraEntropy0, 'hex')
+      const expectedExtraEntropy1 = Buffer.from(f.extraEntropy1, 'hex')
+      const expectedExtraEntropyRand = Buffer.from(f.extraEntropyRand, 'hex')
+      const expectedExtraEntropyN = Buffer.from(f.extraEntropyN, 'hex')
+      const expectedExtraEntropyMax = Buffer.from(f.extraEntropyMax, 'hex')
+
+      const sig = binding.sign(m, d)
+
+      const extraEntropyUndefined = binding.sign(m, d, undefined)
+      const extraEntropy0 = binding.sign(m, d, buf1)
+      const extraEntropy1 = binding.sign(m, d, buf2)
+      const extraEntropyRand = binding.sign(m, d, buf3)
+      const extraEntropyN = binding.sign(m, d, buf4)
+      const extraEntropyMax = binding.sign(m, d, buf5)
+
+      t.same(sig, expectedSig, `sign(${f.m}, ...) == ${f.signature}`)
+      t.same(extraEntropyUndefined, expectedSig, `sign(${f.m}, ..., undefined) == ${f.signature}`)
+      t.same(extraEntropy0, expectedExtraEntropy0, `sign(${f.m}, ..., 0) == ${f.signature}`)
```

Comment:
just realized all the strings use `${f.signature}` oops. I'll fix later when I get to a PC.

Reply:
Also, I might want to add some examples to invalid that have bad extraEntropy.

Reply:
What is bad extra entropy?

Reply:
wrong type.
Buffer wrong size.

etc. etc.

Reply:
ping

Reply:
bad extraEntropy == Not a Buffer. etc.

Making sure type checking is working properly.